packages/core-components/src/lib/unsorted/viz/Chart.svelte

```diff
@@ -191,6 +192,8 @@
 	let columnSummaryArray;
 	let dateCols;
 
+	$: if (data.error) error = data.error.message;
```

Comment:
Maybe switch to a nullish coalesce? This + the `export let data = undefined` is a bit unnerving (even though realistically you wouldn't use a chart w/o `data` supplied

Reply:
Also, should this be outside of the big try/catch block? `error` is reset at the start of that, and it's rerun every time `data` is changed, so it might cancel this assignment out

Reply:
> Also, should this be outside of the big try/catch block? error is reset at the start of that, and it's rerun every time data is changed, so it might cancel this assignment out

I was able to get errors from deeper in the chart and from the querystore to appear properly so I think we're good there